datacube/drivers/postgres/_schema.py

```diff
@@ -27,6 +27,9 @@
     Column('added', DateTime(timezone=True), server_default=func.now(), nullable=False),
     Column('added_by', sql.PGNAME, server_default=func.current_user(), nullable=False),
 
+    # Note that there's an `updated` column added as part of >1.8.1
```

Comment:
So what stops us from just adding new column here? Will this break interop with older db versions?

Reply:
@jeremyh says that it might break queries on DBs that don't have the columns

I figure we should perhaps wait until we have another point release.

Reply:
Yeah, including it will mean that all `select` queries of datasets will include it, which will break on datacubes that have not been updated with the column .

We probably should mention 'why' in our comments -- everyone that reads them will have this question.
Something like:

```suggestion
    # Note that the `updated` column is not included here to maintain backwards-compatibility
    # with pre-1.8.3 datacubes (and it is not used by any internal odc functionality yet anyway)
```